Xamarin.Forms.Core/DependencyService.cs

```diff
@@ -91,7 +91,17 @@ internal static void Initialize(Assembly[] assemblies)
 			// Naive implementation can easily take over a second to run
 			foreach (Assembly assembly in assemblies)
 			{
-				Attribute[] attributes = assembly.GetCustomAttributes(targetAttrType).ToArray();
+				Attribute[] attributes;
+				try
+				{
+					attributes = assembly.GetCustomAttributes(targetAttrType).ToArray();
+				}
+				catch (System.IO.FileNotFoundException)
+				{
+					// Sometimes the previewer doesn't actually have everything required for these loads to work
+					continue;
```

Comment:
Can we at least log a warning for this so someone who's not using the Previewer can see that a problem is occurring (rather than pulling out their hair trying to determine why their custom renderer isn't loading)?